neuralprophet/utils.py

```diff
@@ -987,3 +987,153 @@ def configure_trainer(
     # config["replace_sampler_ddp"] = False
 
     return pl.Trainer(**config), checkpoint_callback
+
+
+def unpack_sliced_tensor(
+    sliced_tensor,
+    n_lags,
+    n_forecasts,
+    max_lags,
+    feature_indices,
+    config_lagged_regressors,
+    config_seasonality,
+):
+    sliced_tensor = sliced_tensor.detach().clone()
```

Comment:
Can we remove this now as you added `contiguous` to the targets?
Or should we keep this, but change to `contiguous`?